src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -10,63 +10,107 @@ class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
   goForward = () => {
     this.props.router.push(this.props.route.pageList[1].path);
   }
+
+  renderSignInMessage = () => {
+    return (
+      <div>
+        <div className="usa-alert usa-alert-info">
+          <div className="usa-alert-body">
+            If you're signed in to your account, the application process can go more smoothly. Here's why:
+            <br/>
+            <ul>
+              <li>We can prefill part of your application based on your account details.</li>
+              <li>You can save your form in progress, and come back later to finish filling it out.</li>
+              <li>You could get your card sooner because we can confirm your Veteran status more quickly.</li>
```

Comment:
@peggygannon just checking that "could" is used here instead of "can" (which is used above) to distinguish the certainty of this statement?

Reply:
@cehsu that was old copy. Current copy has been updated.